nexus/db-queries/src/db/pool.rs

```diff
@@ -140,4 +151,36 @@ impl Pool {
     ) -> anyhow::Result<qorb::claim::Handle<QorbConnection>> {
         Ok(self.inner.claim().await?)
     }
+
+    /// Stops the qorb background tasks, and causes all future claims to fail
+    pub async fn terminate(&self) {
+        let _termination_result = self.inner.terminate().await;
```

Comment:
Should the result be logged?

Reply:
Eh, it can be, but this only matters if terminate is called multiple times